client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+{
+    "common": {
+        "ok": "OK",
+        "submit": "Übernehmen",
+        "cancel": "Abbrechen",
+        "overwrite": "Überschreiben",
+        "continue": "Weiter",
+        "dismiss": "Abbrechen",
+        "prompt": "Hinweis",
+        "warning": "Warnung",
+        "attention": "Achtung!",
+        "confirm": "Bestätigen",
+        "close": "Schließen",
+        "save": "Speichern",
+        "cancelled": "Abgebrochen",
+        "enabled": "Eingeschaltet",
+        "commandOK": "Befehl erfolgreich übertragen!",
+        "commandFailed": "Es ist ein Fehler aufgetreten:",
```

Comment:
Shorter (and maybe less prosaic) alternatives:
"commandOK": "Befehl gesendet",
"commandFailed": "Fehler beim Senden",